src/main/java/hudson/plugins/deploy/DefaultCargoContainerAdapterImpl.java

```diff
@@ -37,22 +38,22 @@
      */
     public void configure(Configuration config, EnvVars envVars, VariableResolver<String> resolver) {
         for(Field f : getClass().getFields()) {
-            setConfiguration(f, config, envVars, resolver);
+            setConfiguration(f, config, envVars);
         }
         for (Method m : getClass().getMethods()) {
-            setConfiguration(m, config, envVars, resolver);
+            setConfiguration(m, config, envVars);
         }
     }
     
-    private void setConfiguration(AccessibleObject ao, Configuration config, EnvVars envVars, VariableResolver<String> resolver) {
+    private void setConfiguration(AccessibleObject ao, Configuration config, EnvVars envVars) {
```

Comment:
I am not 100% convinced that EnvVars are drop-in replacements for the build VariableResolver.  But they're also the modern approach to managing the configuration, so it's probably worth a little risk.